app/lib/screens/identity_verification_screen.dart

```diff
@@ -1265,6 +1266,11 @@ class _IdentityVerificationScreenState
                 ],
               ));
     } on NotEnoughBalance catch (_) {
+      final wallet = (await getPkidWallets())
+          .where((w) => w.type == WalletType.NATIVE)
+          .toList()
+          .first
```

Comment:
what if the wallet is not initialized and the list is empty?

Reply:
Handled.